src/components/UNSTABLE__Shell/PopoverHeader/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file Popover header.
+ * @copyright IBM Security 2020
+ */
+
+import { node } from 'prop-types';
+import React from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+import popoverNamespace from '../Popover';
+
+function PopoverHeader({ children, ...other }) {
+  return (
+    <section
+      className={appendComponentNamespace(popoverNamespace, 'header')}
```

Comment:
Similar suggestion as in `PopoverFooter` -- can we set this up where the namespace in available to exported for tests?

Reply:
Resolving thread in favor of https://github.com/carbon-design-system/ibm-security/pull/285#discussion_r384741145